clients/OwnerClient.ts

```diff
@@ -119,17 +119,17 @@ export class OwnerClient extends StateChannelWallet {
       target: payee,
       amount,
       hashLock: invoice.hashLock,
-      timelock: 0, // todo
+      timelock: BigInt(0), // todo
       updatedState: signedUpdate,
     });
   }
 
   // Create L1 payment UserOperation and forward to intermediary
-  async payL1(payee: string, amount: number): Promise<string> {
+  async payL1(payee: string, amount: bigint): Promise<string> {
     // Only need to encode 'to' and 'amount' fields (i.e. no 'data') for basic eth transfer
     const callData = IAccount.encodeFunctionData("execute", [
       payee,
-      ethers.parseEther(amount.toString()),
+      amount,
```

Comment:
Even though we were passing in a decimal string (`"0.05"`) as the `amount` from the UI, this call to `parseEthers` was converting that string to a `bigInt`.

Now the Wallet UI calls `ethers.ParseEther` itself, so the client can deal with the raw value in wei.